example/config/README.md

```diff
@@ -0,0 +1,249 @@
+        - (Second quantity)
+            - `quantity`: `some_flag` (Since we're specifying `aggregates_imputation` above, a feature-specific imputation rule can be omitted)
+            - `metrics`: 
+                - `max`
+                - `sum`
+
+    - `categoricals`:  Categorical features. The column given can be of any type, but the choices must comparable to that type for equality within SQL The result will be one feature for each choice/metric combination.
+    - (First column)
+        - `column`: Note that we haven't specified a top level `categoricals_imputation` set of rules, so we have to include feature-specific imputation rules for both of our categoricals here.
+        - `imputation`:
+            - `sum`:
+                - `type`: `null_category`
+            - `max`:
+                - `type`: `mean`
+        - `choices`:
+        - `metrics`: `sum`
+    - (Second column)
+        - `column`: `shape` (As with the top-level imputation rules, `all` can be used for the feature-level rules to specify the same type of imputation for all aggregation functions)
+        - `imputation`:
+            - `all`:
+                `type`: `zero`
+        - `choice_query`: `select distinct shape from cool stuff`
+        - `metrics`: 
+            - `sum`
+    - `intervals`: The time intervals over which to aggregate features
+    - `groups`: A list of different columns to separately group by
+
+### Feature Grouping
+define how to group features and generate combinations
+
+- `feature_group_definition`: feature_group_definition allows you to create groups/subset of your features by different criteria. for instance,
+    - `tables`: allows you to send a list of collate feature tables (collate builds these by appending `aggregation_imputed` to the prefix)
+    - `prefix`: allows you to specify a list of feature name prefixes
+
+- `feature_group_strategies`: strategies for generating combinations of groups. available: all, leave-one-out, leave-one-in, all-combinations
+
+### User Metadata
+These are arbitrary keys/values that you can have Triage apply to the metadata for every matrix in the experiment. Any keys you include here can be used in the 'model_group_keys' below. For example, if you run experiments that share a temporal configuration but that use different label definitions (say, labeling building inspections with **any** violation as positive or labeling only building inspections with severe health and safety violations as positive), you can use the user metadata keys to indicate that the matrices from these experiments have different labeling criteria. The matrices from the two experiments will have different filenames (and not be overwritten or inappropriately reused), and if you add the label_definition key to the model group keys, models made on different label definition will have different groups. In this way, user metadata can be used to expand Triage beyond its explicitly supported functionality.
```

Comment:
This example is outdated, as both the matrix metadata and model group keys by default include the label name (as documented above and below). One thing JoCo is doing is including a `matchdatetime` in its user metadata that signifies which set of entity_ids was used to create a given model/model group. That way, we don't accidentally match old predictions with new staged data without looking up the new ids first.